packages/components/socialFeed/NewsFeed/NewsFeedInput.tsx

```diff
@@ -716,29 +716,54 @@ export const NewsFeedInput = React.forwardRef<
                 }}
               >
                 {type === "post" && (
-                  <OmniLink to={{ screen: "FeedNewArticle" }}>
-                    <SecondaryButtonOutline
-                      size="SM"
-                      color={
-                        formValues?.message.length >
-                        SOCIAL_FEED_ARTICLE_MIN_CHARS_LIMIT
-                          ? primaryTextColor
-                          : primaryColor
-                      }
-                      borderColor={primaryColor}
-                      touchableStyle={{
-                        marginRight: layout.spacing_x2,
-                      }}
-                      backgroundColor={
-                        formValues?.message.length >
-                        SOCIAL_FEED_ARTICLE_MIN_CHARS_LIMIT
-                          ? primaryColor
-                          : neutral17
-                      }
-                      text="Create an Article"
-                      squaresBackgroundColor={neutral17}
-                    />
-                  </OmniLink>
+                  <>
+                    <OmniLink to={{ screen: "FeedNewArticle" }}>
+                      <SecondaryButtonOutline
+                        size="SM"
+                        color={
+                          formValues?.message.length >
+                          SOCIAL_FEED_ARTICLE_MIN_CHARS_LIMIT
+                            ? primaryTextColor
+                            : primaryColor
+                        }
+                        borderColor={primaryColor}
+                        touchableStyle={{
+                          marginRight: layout.spacing_x2,
+                        }}
+                        backgroundColor={
+                          formValues?.message.length >
+                          SOCIAL_FEED_ARTICLE_MIN_CHARS_LIMIT
+                            ? primaryColor
+                            : neutral17
+                        }
+                        text="Create an Article"
```

Comment:
I think we can remove everything related to old articles creation in this PR

Reply:
Don't lose time to review, it's a draft ^^'

Reply:
https://github.com/TERITORI/teritori-dapp/pull/1451/commits/271f701c45bbc8a9b93f63def82a5c44eb13bbdf#diff-fb2d69fc97a6e99c7994ad8cc071998739266572c25aca45b6c8553b0a11c954